ATTRIBUTION.md

```diff
@@ -471,7 +501,7 @@ notice, this list of conditions and the following disclaimer.
 copyright notice, this list of conditions and the following disclaimer
 in the documentation and/or other materials provided with the
 distribution.
```

Comment:
Google is an LLC as of 2017, I wonder if the lib we are using is real old?

Reply:
Yep looks like golang.org/x/exp just updated their license recently https://github.com/golang/exp/commit/1d5bc16f04a822c5d762aca81011815d8bb7eb4b